### PR TITLE
v0.2.0: Add Eth Actions (set_allowance, deposit, withdraw, withdraw_to_zero)

### PR DESCRIPTION
**Pull request by BrendanChou**

### CODE REVIEW

File: dydx/client.py [hunk 1]
```diff
@@ -53,6 +92,81 @@ def _put(self, *args, **kwargs):
...
+        if 'from' not in options:
+            options['from'] = self.public_address
+        if 'gasPrice' not in options:
+            options['gasPrice'] = self.web3.eth.gasPrice
+        if 'value' not in options:
+            options['value'] = 0
+        if 'gas' not in options:
+            options['gas'] = method.estimateGas(options)
+
+        print(options)
```
Comment (AntonioJuliano) [inline dydx/client.py]: what is this for?

File: dydx/constants.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+LIMIT_ORDERS_ADDRESS = '0xeb32d60A5cDED175cea9aFD0f2447297C125F2f4'
+SOLO_MARGIN_ADDRESS = '0x1E0447b19BB6EcFdAe1e4AE1694b0C3659614e4e'
+PAYABLE_PROXY_ADDRESS = '0xa8b39829cE2246f89B31C013b8Cde15506Fb9A76'
+WETH_ADDRESS = '0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2'
+DAI_ADDRESS = '0x89d24A6b4CcB1B6fAA2625fE562bDD9a23260359'
+USDC_ADDRESS = '0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48'
+
+MAX_SOLIDITY_UINT = 115792089237316195423570985008687907853269984665640564039457584007913129639935  # noqa: E501
```
Comment (AntonioJuliano) [inline dydx/constants.py]: how does python handle large numbers? Is there something similar to `BigNumber` you have to use?

Comment (BrendanChou) [reply dydx/constants.py]: Both python 2.7 and python 3 can handle arbitrarily-large integers. This is tested in the test suite which uses MAX_SOLIDITY_INT in a test for all versions of python (the ones listed in the readme)

File: dydx/client.py (hunk 1 above)
Comment (BrendanChou) [reply dydx/client.py]: whoops, will remove

**Pull request merged**